### DISCUSSION TITLE
Shared Worker Executors

### DISCUSSION THREAD

**Discussion by MikeEdgar** [strimzi/strimzi-kafka-operator#4916] [General]
While doing some debugging, I noticed based on the number of threads in the JVM that there are two shared worker executor pools used:
- `kubernetes-ops-pool`
- `kubernetes-ops-tool`

My gut reaction was that it's a typo and only `pool` should be created, but I wanted to check before opening an issue.

**Comment by scholzj** [comment]
Nice typo. There seem to be also many sections of the code running blocking executions not using the pool ... they should all use it.

**Comment by scholzj** [comment]
(`pool` is the right one of course)
